components/NavBar/NavBar.tsx

```diff
@@ -125,18 +125,22 @@ export const NavBar: VoidFunctionComponent = () => {
       <Header>
         <HeaderContentContainer>
           <LogoContainer>
-            <Typography><FormattedMessage id="navbar.logo" /></Typography>
+            <Link href="/">
+              <Button variant="ghost" color="secondary">
```

Comment:
Since this is a button used for navigation, I think they should be a real link `a` and not a `button`. I like to think of them as a link styled like a button. :)

With the EDS button, you can add a prop `as="a"` and then it will render as a link, not a button.